src/embed/connection.rs

```diff
@@ -1,12 +1,12 @@
 use super::dispatch::Request;
 use super::types::*;
 use crate::dag;
-use crate::db;
 use crate::fetch;
 use crate::sync;
 use crate::util::rlog;
 use crate::util::rlog::LogContext;
 use crate::util::to_debug;
+use crate::{db, sync::push::push};
```

Comment:
so weird why it would join these two but not the others above (fetch, sync, etc).

Reply:
Yeah. I don't pay attention to these lines... Moving things around manually before fmt makes it more consistent.